collector/hadoop/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM curlimages/curl:7.82.0 as curler
+ARG JMX_JAR_VERSION=v1.14.0
+USER root
+RUN curl -L \
+    --output /opentelemetry-jmx-metrics.jar \
+    "https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/${JMX_JAR_VERSION}/opentelemetry-jmx-metrics.jar"
+
+RUN curl -L -s \
+    "https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.53.0/otelcol-contrib_0.53.0_linux_amd64.tar.gz" | \
+    tar -xvz -C /
+
```

Comment:
Is all of this necessary to use the JMX receiver with the collector? It doesn't seem like it would be. And I think bundling it in the collector obscures the collector's capabilities.

Reply:
@nslaughter Yes, is convenient to run  `--config=/conf/collector.yml` in the same container environment alomg with  the JMX Gatherer`opentelemetry-jmx-metrics.jar`.